readme.md

```diff
@@ -42,22 +42,9 @@ Happy coding!
 
 1) Complete your environment file.
 
-2) Building docker images: 
+2) Building and running docker images: 
 ```bash
-docker-compose up -d
-```
-
-3) Depens on operation system, you need to add permission to some folder: 
-```bash
-chmod 777 -R storage/
-chmod 777 -R public/
```

Comment:
Just in case, are we sure this workaround is no longer needed?

Reply:
:thinking: I thought I had resolved these. I'm testing now and I'm hitting a 500 internal server error when going to http://localhost:8080/api/trips due to apache not being able to write logs to `storage` folder.

I'll re-review this

Reply:
@adalessandro you were right. I've updated the readme at 34b88e085a9398effbdea8cc090b986a99d39fb1